### PR TITLE
Polish: Addressing First Batch of Feedback for MDB.design

### PR DESCRIPTION
**Pull request by brookescarlett**
Addresses the initial batch of feedback from MDB.design

### CODE REVIEW

File: website/components/live-example/LiveExample.tsx [hunk 1]
```diff
@@ -146,7 +154,15 @@ function LiveExample<ComponentProps extends ComponentPropsInterface>({
           );
 
         case Knob.Number:
-          return <Number {...sharedProps} value={props[propName] as number} />;
+          return (
+            <Number
+              {...sharedProps}
+              value={props[propName] as number}
+              min={knobConfig?.min}
+              max={knobConfig?.max}
```
Comment (brookescarlett) [inline website/components/live-example/LiveExample.tsx:162]: Typescript not believing that min, max and step are valid values, but They've been added to the KnobConfigInterface (lines 94-99)

File: packages/syntax/README.md [hunk 2]
```diff
@@ -23,8 +23,7 @@ npm install @leafygreen-ui/syntax
 ```js
 import Syntax from '@leafygreen-ui/syntax';
 
-const codeSnippet = `
-function greeting(entity) {
+const codeSnippet = `dfunction greeting(entity) {
```
Comment (lazytype) [inline packages/syntax/README.md:26]:
```suggestion
const codeSnippet = `function greeting(entity) {
```

File: website/pages/component/popover/example.tsx [hunk 3]
```diff
@@ -73,11 +72,7 @@ function DefaultExample({
   const [active, setActive] = useState(false);
   return (
     <div>
-      <Button
-        onClick={() => setActive(!active)}
-        variant="primary"
-        className={containerStyle}
-      >
+      <button onClick={() => setActive(!active)} className={containerStyle}>
```
Comment (lazytype) [inline website/pages/component/popover/example.tsx:75]: won't need this change anymore if https://github.com/mongodb/leafygreen-ui/pull/732 gets merged

File: website/pages/component/toast/example.tsx [hunk 4]
```diff
@@ -17,6 +17,9 @@ const knobsConfig: KnobsConfigInterface<{
     type: 'number',
     default: 1,
```
Comment (lazytype) [inline website/pages/component/toast/example.tsx:18]: maybe change the default to 0.5

File: website/pages/component/popover/example.tsx (hunk 3 above)
Comment (brookescarlett) [reply website/pages/component/popover/example.tsx:75]: True!

File: website/pages/component/toast/example.tsx (hunk 4 above)
Comment (brookescarlett) [reply website/pages/component/toast/example.tsx:18]: 👍

File: packages/syntax/README.md (hunk 2 above)
Comment (brookescarlett) [reply packages/syntax/README.md:26]: 👍

File: website/components/live-example/LiveExample.tsx (hunk 1 above)
Comment (brookescarlett) [reply website/components/live-example/LiveExample.tsx:162]: @lazytype any thoguhts on what I'm doing wrong here?

File: website/pages/component/popover/example.tsx (hunk 3 above)
Comment (brookescarlett) [reply website/pages/component/popover/example.tsx]: Reverted!

File: website/components/live-example/LiveExample.tsx (hunk 1 above)
Comment (lazytype) [reply website/components/live-example/LiveExample.tsx:162]: Most of the `*ConfigInterface` types don't have `min`, `max`, and `step` properties, so attempting to `Extract` the properties from them wouldn't have any effect. This `switch` statement also doesn't have any narrowing effect, as you can tell from the need to assert `knobConfig?.options as Array<string>` below in the `Knob.Select` case.

Comment (brookescarlett) [reply website/components/live-example/LiveExample.tsx:162]: 👍

**Review by lazytype — APPROVED** [review]

**Pull request merged**